config/marklogic-sink.properties

```diff
@@ -1,113 +1,130 @@
-# Kafka-specific properties
+# Properties required for every Kafka connector
 
+# The name of the connector
 name=marklogic-sink
+
+# The fully-qualified name of the connector class
 connector.class=com.marklogic.kafka.connect.sink.MarkLogicSinkConnector
 
-# Should only need one task since it's using a WriteBatcher, which is multi-threaded
+# The maximum number of concurrent tasks
 tasks.max=1
 
-# Topics to consume from [comma separated list for multiple topics]
+# Comma-separated list of topics to subscribe to
 topics=marklogic
 
 
-# MarkLogic connector-specific properties
+# Properties defined by the MarkLogic Kafka connector
 
-# A MarkLogic host to connect to. The connector uses the Data Movement SDK, and thus it will connect to each of the
-# hosts in a cluster.
+# Required; a MarkLogic host to connect to. By default, the connector uses the Data Movement SDK, and thus it will
+# connect to each of the hosts in a cluster.
 ml.connection.host=localhost
 
-# The port of a REST API server to connect to.
+# Required; the port of a REST API server to connect to
 ml.connection.port=8000
 
-# Optional - the name of a database to connect to. If your REST API server has a content database matching that of the
-# one that you want to write documents to, you do not need to set this.
-ml.connection.database=Documents
+# Required; the authentication scheme used by the server defined by ml.connection.port; either 'DIGEST', 'BASIC', 'CERTIFICATE', 'KERBEROS', or 'NONE'
+ml.connection.securityContextType=DIGEST
 
-# Optional - set to "gateway" when using a load balancer, else leave blank.
-# See https://docs.marklogic.com/guide/java/data-movement#id_26583 for more information.
-ml.connection.type=
+# MarkLogic username for 'DIGEST' and 'BASIC' authentication
+ml.connection.username=
 
-# Either DIGEST, BASIC, CERTIFICATE, KERBEROS, or NONE
-ml.connection.securityContextType=DIGEST
+# MarkLogic password for 'DIGEST' and 'BASIC' authentication
+ml.connection.password=
+
+# Path to PKCS12 file for 'CERTIFICATE' authentication
```

Comment:
Commenting out anything not used in this file. The README should be the authoritative source on properties and default values. This file instead has some additional values set to simplify testing the connector.